tools/make/kube.mk

```diff
@@ -20,41 +20,9 @@ ifndef ignore-not-found
 endif
 
 .PHONY: kube-install
-kube-install: manifests kustomize ## Install CRDs into the Kubernetes cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build pkg/provider/kubernetes/config/crd | kubectl apply -f -
+kube-install: manifests $(tools/kustomize) ## Install CRDs into the Kubernetes cluster specified in ~/.kube/config.
+	$(tools/kustomize) build pkg/provider/kubernetes/config/crd | kubectl apply -f -
 
 .PHONY: kube-uninstall
-kube-uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
-	$(KUSTOMIZE) build pkg/provider/kubernetes/config/crd | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
-
-##@ Kubernetes Build Dependencies
```

Comment:
any specific reason to rm the current way ?

Reply:
consistency

Reply:
thanks for the detailed reasoning, im still not convinced that this is the better approach for this project, and the existing one that is similar the kubebuilder approach is much simpler to read through and maintain.
apart from this change, the PR LGTM, thanks for the multiple rebases, smaller PRs for specific areas should hopefully reduce such conflicts